ECMAScript/react/README.md

```diff
@@ -74,13 +93,10 @@ const MyComponent = () => {
   const { session, importSession, authenticateSession, revokeSession } = useQuilttSession()
 
   useEffect(() => {
-    // Import session from local storage or any other source
-    importSession()
-
-    // Authenticate session with your backend if necessary
-    authenticateSession()
+    // Import session from local storage or an API call
+    importSession('{SESSION_TOKEN}')
 
-    // Clean up session when component unmounts
+    // Revoke and clean up session when component unmounts
     return () => {
       revokeSession()
     }
```

Comment:
This is bug waiting for someone to copy and paste it. Revoking the session when the component is torn down is not the same as someone clicking logout.
```suggestion
```

Reply:
Are you saying we shouldn't revoke session when the connector unmounted?

Reply:
Removed. I documented `revokeSession` with a log out button in another example.